Notation.Plugin.AzureKeyVault/KeyVault/KeyVaultClient.cs

```diff
@@ -190,7 +192,15 @@ public async Task<X509Certificate2Collection> GetCertificateChainAsync()
             {
                 case "application/x-pkcs12":
                     // If the secret is a PKCS12 file, decode the base64 encoding
-                    chain.Import(Convert.FromBase64String(secretValue), "", X509KeyStorageFlags.EphemeralKeySet);
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                    {
+                        // macOS doesn't support non-encrypted MAC
+                        chain.Import(Pkcs12.RemoveMac(Convert.FromBase64String(secretValue)), null, X509KeyStorageFlags.DefaultKeySet);
+                    }
```

Comment:
The PR check complains about the unit test: #L196 - L199 were not covered by tests.

Can you fix it by any chance?

Reply:
BTW, we need to improve the test workflow `build` to `build-windows`, `build-linux`, `build-mac` in the future.

Reply:
Added unit test.

Reply:
https://github.com/Azure/notation-azure-kv/issues/125
Added an issue